api_examples.py

```diff
@@ -0,0 +1,70 @@
+import json
+
+
+class APIExampleMixin:
+    """A TestCase mixin that allows access to API by Example retrieval methods."""
+
+    SERVER_NAME = 'localhost:8000'
+
+    def create_request(self, *args, **kwargs):
+        return super().create_request(*args, SERVER_NAME=self.SERVER_NAME, **kwargs)
```

Comment:
Following investigation, I think there's a better way to handle this in our projects, so I've removed this method.